diagnostics-app/src/app/rma/test-viewer/json-viewer/json-viewer.component.ts

```diff
@@ -11,18 +11,6 @@ export class JsonViewerComponent {
   @Input({required: true}) output!: any;
   @Input({required: true}) level!: number;
 
-  // This function is used for maintaining the attributes' original order.
-  /* eslint-disable @typescript-eslint/no-unused-vars */
-  public originalOrder(
-    a: KeyValue<number, string>,
-    b: KeyValue<number, string>,
-  ): number {
-    return 0;
-  }
-  /* eslint-enable @typescript-eslint/no-unused-vars */
-
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public isPrimitiveType(a: any) {
-    return typeof a !== 'object';
-  }
+  public originalOrder = originalOrder;
+  public isPrimitiveType = isPrimitiveType;
```

Comment:
nit: Since there is no caller in this ts file, maybe we should add some comments for future reference, e.g. `Helper functions for html/frontend/ng files`.

Ditto to others.

Or we can have some comments in `app-utils.ts` instead.

Reply:
Done.